packages/titanium-single-select/src/titanium-single-select.ts

```diff
@@ -364,7 +372,8 @@ export class TitaniumSingleSelectElement extends LitElement {
       border-top: 0;
       border-bottom-left-radius: 4px;
       border-bottom-right-radius: 4px;
-      box-shadow: 0 4px 6px 0 rgba(32, 33, 36, 0.28);
+      margin-top: -1px;
+      box-shadow: 0 6px 6px 0 rgba(32, 33, 36, 0.28);
```

Comment:
Pushed box-shadow down for the same problem as above.  It was visible between the two elements.